src/Functions/userAuth.ts

```diff
@@ -0,0 +1,27 @@
+import dotenv from 'dotenv';
+dotenv.config();
+import jwt from 'jsonwebtoken';
+const secret= process.env.SECRET_PHRASE
+
+export async function CreateTokenLogin(payload: any, expiresIn:string){
```

Comment:
```suggestion
export async function createTokenLogin({ payload, expiresIn}: {payload: {username: string}, expiresIn: string ){
```
- Pas besoin de majuscule
- Dans mes appels à fonction souvent je passe des objets, ça permet de spécifier les noms quand tu les appelles
- Specifie ton type de payload `{id: string}` ou `{username: string}` et laisse pas de any
-  Pour le type de expiresIn, string c'est la version facile sinon utilise [ca](https://www.typescriptlang.org/docs/handbook/utility-types.html#parameterstype) pour le type de expiresIn (mais string ça va très bien dans un premier temps